docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -615,7 +615,7 @@ Enables the GPIO output associated with air-blast, set by the operator during th
 M7
 ```
 
-Enables the GPIO output associated with unpressurized coolant, set by the operator during the configuration wizard. If air blast (`M7.1`) is not already enabled, then this will be enabled prior to activating the coolant output.
+Enables the GPIO output associated with unpressurized coolant, set by the operator during the configuration wizard. If air blast (`M7.1`) is not already enabled, then this will be enabled before activating the coolant output.
```

Comment:
```suggestion
Enables the general-purpose output associated with unpressurized coolant, set by the operator during the configuration wizard. If air blast (`M7.1`) is not already enabled, then this will be enabled before activating the coolant output.
```